src/MCPClient/tests/test_antivirus_clamdscan.py

```diff
@@ -68,6 +69,10 @@ def test_clamdscanner_scan(mocker, settings):
     FOUND_RET = ('FOUND', 'Eicar-Test-Signature')
 
     def patch(scanner, ret=OKAY_RET, excepts=False):
+        """Patch the scanner function and enable testing of exceptions raised
+        by clamdscanner that we want to control. excepts can take an argument
+        of True to pass a generic exception. excepts can also take an exception
+        as an argument for better granularity."""
```

Comment:
Nitpick: the final `"""` should be on its own line, cf. https://www.python.org/dev/peps/pep-0008/#documentation-strings.